### ISSUE TITLE
Error generated when changing episode quailty

### ISSUE THREAD

**Issue by johnvick** [pymedusa/Medusa#6783]
Branch: master
Commit: 6a21a6001b10827d14d450135bae61729b01edf1
Version: 0.3.2
Database: 44.14
Python Version: | 3.5.3 (default, Sep 27 2018, 17:25:39) [GCC 6.3.0 20170516]
SSL Version: | OpenSSL 1.1.0j 20 Nov 2018
OS: | Linux-4.14.66+-armv7l-with-debian-9.9
Locale: | en_GB.UTF-8

Change episode quality leads to white screen and this error in logs:
```
2019-06-06 15:22:55 ERROR    Thread-61 :: [6a21a60] Exception generated: invalid literal for int() with base 10: 'Change status to:'
Traceback (most recent call last):
  File "/home/john/medusa/medusa/server/web/core/base.py", line 261, in async_call
    result = function(**kwargs)
  File "/home/john/medusa/medusa/server/web/home/handler.py", line 1592, in setStatus
    status = int(status)
ValueError: invalid literal for int() with base 10: 'Change status to:'
```
The episode quality is changed OK.

John

**Comment by sharkykh** [comment]
Oops. My bad.
Everything still works as intended.
It just generates an error because the "Change status to:" option didn't have a value (and I assumed wrong).
Will be fixed with #6784

**Comment by medariox** [comment]
Fixed with https://github.com/pymedusa/Medusa/pull/6784
